### PR TITLE
Add chaperone API

### PR DESCRIPTION
**Pull request by tmfink**

### CODE REVIEW

File: src/chaperone.rs [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        let is_ok = unsafe { self.0.GetPlayAreaRect.unwrap()(&mut play_area_rect) };
+        if is_ok {
+            Some(play_area_rect)
+        } else {
+            None
+        }
+    }
+
+    // Do not understand what this does
+    /// Reload chaperone info
```
Comment (Ralith) [inline src/chaperone.rs]: This probably rereads config files, or something similar.

File: src/chaperone.rs [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    fn reload_info(&mut self) {
+        unsafe { self.0.ReloadInfo.unwrap()() };
+    }
+
+    // Do not understand what this does
+    /// Set scene color
```
Comment (Ralith) [inline src/chaperone.rs]: The color of the chaperone wireframe, perhaps?

Comment (tmfink) [reply src/chaperone.rs]: That did not work with my Vive; I tried the reload afterward but still no dice.

**Comment by tmfink** [comment]
Should I remove the "do not understand" comments and just make the methods public? I was just being cautious about exposing API that we do not understand.

**Comment by Ralith** [comment]
Oh, I actually missed that they weren't public. I'd just remove them entirely, in that case; in the unlikely event that someone comes along wanting them before this whole API is obsoleted by OpenXR, they can add and document them properly.

**Pull request merged**